R/get_cdc_data.R

```diff
@@ -0,0 +1,41 @@
+#' Retrieve CDC growth chart metadataset
+#'
+#' Retrieves CDC growth chart metadataset
+#'
+#' @param type Type of Chart
+#'
+#' A character is expected amongst: "height", "weight", "bmi"
+#'
+#' @return The appropriate growth chart from the CDC data
+#'
+#' @details
+#'
+#' Each data frame will be denoted as below:
+#' \describe{
+#'   \item{\code{SEX}}{double sex where 1 = male, 2 = female}
+#'   \item{\code{AGE}}{double age in months}
+#'   \item{\code{L}}{double Box-Cox transformation for normality}
+#'   \item{\code{M}}{double Median}
+#'   \item{\code{S}}{double Coefficient of variation}
+#' }
+#'
+#' from [cdc websites](https://www.cdc.gov/growthcharts/percentile_data_files.htm)
+#'
+#' @family metadata
+#' @keywords metadata
+#'
+#' @export
+#'
+#' @examples
+#' get_cdc_data(type = "bmi") |> head()
```

Comment:
Don't we usually still use `%>%` for admiral pipe convention?

Reply:
With R 4.4 coming in a month, our support going back 3 major versions would include 4.1 where the base pipe was introduced, I figured if we ever wanted to adopt native pipe, its easier to start now than swap later, I can switch it though

Reply:
there was a discussion in admiral repo on this before that i saw and it was deemed not worth to change yet as there were some cases Stefan pointed out where `%>%` works better. i forget the details. please do update